src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,130 @@
+.clock {
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  margin: auto;
+
+  @include watch-size($watch-size);
+
+  background-color: beige;
+  border-radius: 100%;
+  border: 10px solid $clock-color;
+
+  /* box-shadow: 40px 5px 2px #787d7e; */
+
+  &__top {
+    position: absolute;
+    left: 0;
+    right: 0;
```

Comment:
But why do you need to make it 100% width for every element? Try to do it with a shorter solution.

Reply:
fixed